tests/e2e/client_internal_test.go

```diff
@@ -60,8 +60,8 @@ type OfcirAcquire struct {
 	Type         string
 }
 
-func (c *OfcirClient) Acquire() (*OfcirAcquire, error) {
-	destUrl := fmt.Sprintf("%s/v1/ofcir?type=host", c.baseUrl)
+func (c *OfcirClient) Acquire(cirtype string) (*OfcirAcquire, error) {
+	destUrl := fmt.Sprintf("%s/v1/ofcir?type="+cirtype, c.baseUrl)
```

Comment:
Not sure about this change, why it was required?

Reply:
Because type=host was hardcoded, one of the tests now needs type=host,host2